### ISSUE TITLE
[Feature] Phone Notifications

### ISSUE THREAD

**Issue by icefields** [izivkov/GShockAPI#3]
Hello, 
I own a GBD-200, which despite not being in the list of supported watches it still works with most commands. 
The Move app is able to read notifications and send them to the watch, I would love to have something like that in the Casio G-Shock Smart Sync app or in this API.
Thanks

**Comment by izivkov** [comment]
I have been wanting to incorporate the Move app into this one for quite a while, but I do not have this watch. I am now focusing on reworking the app with a modern Material 3 interface, which will take about a month. After that, I will try to get somebody to lend me this watch, or donate it to the project. I cannot justify buying every Casio watch since I am not a collector. ;-)

Anyway, thanks for reporting.
Regards

**Comment by icefields** [comment]
I was wondering if this API would be able to send the notifications to the watch, I think other watches you have in the list can also receive notifications

**Comment by izivkov** [comment]
I see you are a developer. If you would like to contribute to the project and add the fitness feature to the watch, I will welcome this.

**Comment by izivkov** [comment]
No, not at the moment. The watch has to be always connected for that, like
the ECB-30, but I would live to add notifications in the future

**Comment by icefields** [comment]
@izivkov yeah I was thinking about either contributin to yout code, or making a separate app or plugin specifically for notifications. On the android side I know exactly what to do, I can capture notifications from any application, filter them, and so on. I just don't have enough knowledge on how to send them to the watch via Bluetooth, that's why I was looking into your API, hoping you had something in place to send a payload to the watch that the watch would see as a watch notification.
At the moment my open source project Power Ampache 2 is taking most of my time, so I don't really have much time to study the Bluetooth interface and figure this out by myself.

**Comment by izivkov** [comment]
@icefields 
OK, yes, figuring out the forfmat of the data sent to the watch is a tediius process. It involves looking ar WireShark logs, and then figuring out how they encode data. If I had the watch, I would probobly try it, but without the watch is impossible. 

Anyway, if you create your oun project using the API, let me know so i can see how you handle messages.
Regards,
Ivo

**Comment by icefields** [comment]
@izivkov I initially assumed that some of the watches you support already had the notification feature. Regardless, I'll definitely keep you updated if I find time to move forward with the project.

**Comment by izivkov** [comment]
These watches are partly supported, in a sense that  the app can run on
them. Not all features are supposed

**Comment by izivkov** [comment]
BTW, I am rewriting the app in Jetpack Compose from the ground up. I wish I
started that way. Way cleaner and easy to maintain. Hope to have a first
version done in a couple of weeks for testing only.

**Comment by icefields** [comment]
I was looking at your code to see how can I contribute, and maybe involve my designer too (just a thought for now, as I said Power Ampache 2 is taking most of our free time), I noticed a few things, mostly on the architectural side of things, if you're interested we can chat in private.

**Comment by izivkov** [comment]
Sure, I'd be happy to chat.

**Comment by icefields** [comment]
thanks. Yes, I'm talking about the app, I haven't checked the code for the API yet. 
I will send you an email later, pretty busy day today.
